truffle.js

```diff
@@ -0,0 +1,22 @@
+require('babel-register')
+require('babel-polyfill')
```

Comment:
These aren't great here btw, I had to comment them out. Not sure why but `truffle migrate` fails because apparently the env where Truffle runs the migrations can't find those from node-modules (though they clearly are there). Maybe it's in my env, I don't know; but on the other hand, what's the point of those requires?